src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/RecordRemissionDecisionPreparer.java

```diff
@@ -114,6 +114,11 @@ private boolean isRemissionExists(Optional<RemissionType> remissionType) {
         return remissionType.isPresent() && remissionType.get() != RemissionType.NO_REMISSION;
     }
 
+    private boolean isRemissionExistsAip(Optional<RemissionOption> remissionOption) {
+        boolean isDlrmFeeRemission = featureToggler.getValue("dlrm-fee-remission-feature-flag", false);
+        return remissionOption.isPresent() && remissionOption.get() != NO_REMISSION && isDlrmFeeRemission;
```

Comment:
When you select No_remission_option you redirected to another Page which contains other possible options which still brings us to the Remission. 

```
function appealHasRemissionOption(application: AppealApplication) {
  return [
    'asylumSupportFromHo',
    'feeWaiverFromHo',
    'under18GetSupportFromLocalAuthority',
    'parentGetSupportFromLocalAuthority'
  ].includes(application.remissionOption) ||
    'noneOfTheseStatements' === application.remissionOption &&
    ('wantToApply' === application.helpWithFeesOption || 'alreadyApplied' === application.helpWithFeesOption)
    && application.helpWithFeesRefNumber;
}

function appealHasNoRemissionOption(application: AppealApplication) {
  return 'noneOfTheseStatements' === application.remissionOption && 'willPayForAppeal' === application.helpWithFeesOption;
}
```

Check if it is ok with your conditions.

Reply:
Have added the following to the isRemissionExistsAip method to consider this scenario:

`helpWithFeesOption.isPresent() && helpWithFeesOption.get() == WANT_TO_APPLY`